rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book
+
+  def initialize(person, date, book)
+    @person = person
+    @date = date
+    @book = book
```

Comment:
- [ ] Good job working on this but please kindly ensure that the rental is adding itself to the @book achieving the 'belongs-to' association
- [ ] Also ensure that the rental is adding itself to the @person achieving the 'belongs-to' association.

Hint:
`book.rentals << self`